nupic/regions/TPRegion.py

```diff
@@ -506,6 +517,18 @@ def _compute(self, inputs, outputs):
       size = activeLearnCells.shape[0] * activeLearnCells.shape[1]
       outputs['lrnActiveStateT'][:] = activeLearnCells.reshape(size)
```

Comment:
@akhilaananthram is this the same field semantically? If so, we should remove either of them: `lrnActiveStateT vs. activeCells`. Or what is the difference?
https://github.com/numenta/nupic/pull/2206/files#diff-7cf0d542d5dacba939c229573fc267e5R528